src/main/java/at/hannibal2/skyhanni/config/features/garden/pests/StereoHarmonyConfig.java

```diff
@@ -0,0 +1,53 @@
+package at.hannibal2.skyhanni.config.features.garden.pests;
+
+import at.hannibal2.skyhanni.config.FeatureToggle;
+import at.hannibal2.skyhanni.config.core.config.Position;
+import com.google.gson.annotations.Expose;
+import io.github.notenoughupdates.moulconfig.annotations.ConfigEditorBoolean;
+import io.github.notenoughupdates.moulconfig.annotations.ConfigOption;
+import io.github.notenoughupdates.moulconfig.observer.Property;
+
+public class StereoHarmonyConfig {
+
+    @Expose
+    @ConfigOption(
+        name = "Enabled",
+        desc = "Shows a display of what pest is being boosted by your vinyl."
+    )
+    @ConfigEditorBoolean
+    @FeatureToggle
+    public boolean displayEnabled = true;
+
+    @Expose
+    @ConfigOption(
+        name = "Always Show",
+        desc = "Always show the display."
+    )
+    @ConfigEditorBoolean
+    public boolean alwaysShow = false;
```

Comment:
Please indicate that this display normally only shows while farming.
Maybe something like this:
![image](https://github.com/hannibal002/SkyHanni/assets/24389977/7e602cb3-953e-4a44-ad4c-41b340939384)